projects/gfs/index.js

```diff
@@ -0,0 +1,43 @@
+const sdk = require("@defillama/sdk");
+const { getChainTransform } = require("../helper/portedTokens");
+const { getBlock } = require("../helper/getBlock");
+const { sumSingleBalance } = require("@defillama/sdk/build/generalUtil");
+const { sumLPWithOnlyOneToken } = require("./../helper/unwrapLPs");
+const BigNumber = require("bignumber.js");
+
+const iotx = "0x6fb3e0a217407efff7ca062d46c26e5d60a14d69";
+const wiotx = "0xA00744882684C3e4747faEFD68D283eA44099D03";
+const liquidityPool = "0x19f3cb6a4452532793d1605c8736d4a94f48752c";
+
+const tvl = async (timestamp, block, chainBlocks) => {
+  let chain = "iotex";
+  block = await getBlock(timestamp, chain, chainBlocks);
+  let balances = { "iotex": 0 };
+
+  const [balanceOfTokenListedInLP] = await Promise.all([
+    sdk.api.erc20.balanceOf({
+      target: wiotx,
+      owner: liquidityPool,
```

Comment:
Sorry, but you cant add entire liquidity pool as tvl, you can only count LP staked by the users on your staking contract and as 'pool2' not under 'tvl'

Reply:
Oh, i have been updated the code, review again pls